packages/oui-clipboard/src/clipboard.html

```diff
@@ -0,0 +1,12 @@
+<div class="oui-clipboard">
+    <div class="oui-clipboard__container">

```

Comment:
Why not use `oui-input-group oui-input-group_button` ? We could even go with a new modifier if necessary (eg. `oui-input-group_clipboard`)